cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -459,7 +552,10 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 			if outMember.Type.IsAssignable() {
 				sw.Do("out.$.name$ = $.outType|raw$(in.$.name$)\n", args)
 			} else {
-				if g.convertibleOnlyWithinPackage(m.Type, outMember.Type) {
+				if g.preexists(m.Type, outMember.Type) {
+					args["function"] = g.preexisting[conversionType{m.Type, outMember.Type}]
```

Comment:
args are reused only within a single member, and generally there is only one non-trivial line for every member. So I think it's fine to use as is.